api_v3/services/UserEntryService.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * @service userEntry
+ * @package api
+ * @subpackage services
+ */
+class UserEntryService extends KalturaBaseService {
+
+	public function initService($serviceId, $serviceName, $actionName)
+	{
+		parent::initService($serviceId, $serviceName, $actionName);
+	}
+
+	/**
+	 * Adds a user_entry to the Kaltura DB.
+	 *
+	 * @action add
+	 * @param KalturaUserEntry $userEntry
+	 * @return KalturaUserEntry
+	 */
+	public function addAction(KalturaUserEntry $userEntry)
+	{
+		$dbUserEntry = $userEntry->toInsertableObject(null, array('type'));
+		$userId = $userEntry->userId;
+		if ($userId == 0)
+		{
+			$userId = kCurrentContext::$ks_kuser;
+		}
+		$dbUserEntry->setKuserId($userId);
```

Comment:
Please implement all these setters inside the KalturaUserEntry object.
user id and status in the toInsertableObject and type in the constructor.